src/webaudio.js

```diff
@@ -257,6 +257,7 @@ WaveSurfer.WebAudio = {
         this.gainNode.disconnect();
         this.scriptNode.disconnect();
         this.analyser.disconnect();
+        this.ac.close();
```

Comment:
If you don't want to remove the `ac` parameter now, we could track when it's passed externally (vs created internally) and close or not close based on whether we created it.